setup.cfg

```diff
@@ -10,15 +10,36 @@ long_description_content_type = text/markdown
 packages = find:
 python_requires = >=3.8
 install_requires =
-    numpy
-    scipy
-    tqdm
-    jax
-    jaxlib
+    jax[cpu]
```

Comment:
Is this the correct way to handle jax as a dependency? As in, what it someone prefers `jax[cuda]`?

Reply:
Hmmmm I didnt think about this, never considered it important thus far :smile: But you are right.

Would you prefer leaving it out here, write "we don't include jax as a dependency, because you need to install your own" in the readme, and move a jax[cpu] dependency to the tests in tox (we need something like this to make the CI go).

Reply:
I made an according update. Let me know what you think.

Reply:
Looks good! I mean I'm not quite sure how jax should be handled, but e.g. blackjax does it the same way (as in "install jax yourself in your preferred way"), so I assume this is a good way to handle it :)